witchcraft/server_run.go

```diff
@@ -115,6 +120,19 @@ func newTLSConfig(serverConfig config.Server, useSelfSignedServerCertificate boo
 	return tlsConfig, nil
 }
 
+func serverClientCAFiles(tlsCertProvider tlsconfig.TLSCertProvider, ignoreMissingClientCAFiles bool, clientCAFiles ...string) tlsconfig.ServerParam {
+	clientCAsParam := tlsconfig.ServerClientCAFiles(clientCAFiles...)
+	if !ignoreMissingClientCAFiles {
+		return clientCAsParam
+	}
+	_, err := tlsconfig.NewServerConfig(tlsCertProvider, clientCAsParam)
```

Comment:
This feels a bit hacky, but I wasn't sure how else to determine if there's any errors with the current set of provided file without making contributions to other vendor libraries (like `ServerClientCAs` in tlsconfgi_server.go). Would appreciate any suggestions/improvements.